packages/fileeditor/src/widget.ts

```diff
@@ -242,11 +161,12 @@ export class FileEditor extends Widget {
     );
   }
 
-  private editorWidget: FileEditorCodeWrapper;
+  private editorWidget: CodeEditorWrapper;
   public model: CodeEditor.IModel;
   public editor: CodeEditor.IEditor;
   protected _context: DocumentRegistry.Context;
   private _mimeTypeService: IEditorMimeTypeService;
+  private _ready = new PromiseDelegate<void>();
```

Comment:
Let's fix the visibility (`context` has public getter so no need to have `_context` protected as it should not be set outside the constructor) and follow the naming convention (`editorWidget` -> `_editorWidget`):

```suggestion
  model: CodeEditor.IModel;
  editor: CodeEditor.IEditor;
  private _context: DocumentRegistry.Context;
  private _editorWidget: CodeEditorWrapper;
  private _mimeTypeService: IEditorMimeTypeService;
  private _ready = new PromiseDelegate<void>();
```